### PR TITLE
Infrastructure Architecture navigation bar

### PR DESCRIPTION
**Pull request by Guaris**
### Summary
IA navbar work

Not adding redirects at this point. 

### Testing
Click the navigation bar
-->

### CODE REVIEW

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 1]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /kong-production/licenses/report
+      - text: Networking
+        items:
+          - text: Default Ports
+            url: /kong-production/networking/default-ports
+          - text: DNS considerations
+            url: /kong-production/networking/dns-considerations
+          - text: Network and Firewall
+            url: /kong-production/networking/network
+      - text: Kong.conf Configuration File
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:132]:
```suggestion
      - text: Kong Configuration File
```
Could just be a personal preference and I know we have "kong.conf Configuration File" in the Google Sheet. Seeing "kong.conf" was just kind of jarring in the sidebar nav.

File: src/gateway/kong-manager/get-started/services-routes.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+title: Services and Routes
```
Comment (cloudjumpercat) [inline src/gateway/kong-manager/get-started/services-routes.md:2]:
```suggestion
title: Configuring Services and Routes
```
Just a suggestion to distinguish this from Understanding Kong: Services or Understanding Kong: Routes. I think having the name as "Services and Routes" in the nav is okay though, just thinking of Google searches to find this page.

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 3]
```diff
@@ -11,504 +11,529 @@ items:
...
-        url: /developer-portal/helpers/cli
-
-  - title: Monitor
-    icon: /assets/images/icons/documentation/icn-vitals.svg
-    items:
-      - text: Kong Vitals
-        url: /vitals
+          - text: Services and Routes
+            url: /kong-manager/get-started/services-routes
+          - text: Rate Limiting
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:298]: For some reason, this page is redirecting to the "Services and Routes" page above it for me.

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 4]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /kong-manager/get-started/rate-limiting
+          - text: Proxy Caching
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:300]: For some reason, this page is redirecting to the "Services and Routes" page above it for me.

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 5]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /kong-manager/authentication/sessions
+          - text: Role Based Access Control
+            items:
+              - text: Overview
+                url: /kong-manager/authentication/rbac/
+              - text: Add a Role
+                url: /kong-manager/authentication/rbac/add-role
+              - text: Add a User
+                url: /kong-manager/authentication/rbac/add-user
+              - text: Add a Admin
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:332]:
```suggestion
              - text: Add an Admin
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 6]
```diff
@@ -11,504 +11,529 @@ items:
...
+                url: /kong-manager/authentication/rbac/add-admin
+          - text: Mapping Ldap Service Directory groups to Roles
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:334]:
```suggestion
          - text: Mapping LDAP Service Directory groups to Roles
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 7]
```diff
@@ -11,504 +11,529 @@ items:
...
+      - text: Networking
+        url: /kong-manager/networking
+      - text: Sending email
+        url: /kong-manager/email
+  - title: Develop Custom Plugins
     icon: /assets/images/icons/documentation/icn-references-color.svg
     items:
-      - text: Admin API
-        url: /admin-api
+      - text: Introduction
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:343]: Should this be "Overview" to match other sections?

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 8]
```diff
@@ -11,504 +11,529 @@ items:
...
+        url: /admin-api/rbac/reference
+        items:
+          - text: RBAC Reference
+            url: /admin-api/rbac/reference
+          - text: RBAC Examples
+            url: /admin-api/rbac/examples
+      - text: Admins
+        url: /admin-api/admins/reference
+        items:
+          - text: API Reference
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:505]:
```suggestion
          - text: Admins Reference
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 9]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /admin-api/admins/reference
+          - text: Examples
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:507]:
```suggestion
          - text: Admins Examples
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 10]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /admin-api/admins/examples
+      - text: Developers
+        url: /admin-api/developers/reference
+      - text: Consumer Groups
+        url: /admin-api/consumer-groups/reference
+        items:
+          - text: API Reference
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:514]:
```suggestion
          - text: Consumer Groups Reference
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 11]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /admin-api/consumer-groups/reference
+          - text: Examples
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:516]:
```suggestion
          - text: Consumer Groups Examples
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 12]
```diff
@@ -11,504 +11,529 @@ items:
...
+            url: /admin-api/consumer-groups/examples
+      - text: Event Hooks
+        url: /admin-api/event-hooks/reference
+        items:
+          - text: Event Hooks Reference
+            url: /admin-api/event-hooks/reference
+          - text: Examples
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:523]:
```suggestion
          - text: Event Hooks Examples
```

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 13]
```diff
@@ -11,504 +11,529 @@ items:
...
+  - title: Reference
+    icon: /assets/images/icons/documentation/icn-references-color.svg
+    items:
+      - text: Configuration Reference
+        url: /reference/configuration
+      - text: Kong Gateway Nginx Directives
+        url: /reference/nginx-directives
+      - text: CLI Reference
+        url: /reference/cli
+      - text: Rate Limiting Library
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:536]: This is pointing to the same page as "Configuring sliding-window". Not sure if that's intentional.

File: app/_data/docs_nav_gateway_3.0.x.yml [hunk 14]
```diff
@@ -11,504 +11,529 @@ items:
...
+        url: /kong-plugins/rate-limiting/algorithms/rate-limiting
+      - text: Performance Testing Framework Reference
```
Comment (cloudjumpercat) [inline app/_data/docs_nav_gateway_3.0.x.yml:538]: This is pointing to the same page as "Kong Performance Testing". Not sure if that's intentional.

File: src/gateway/kong-manager/get-started/services-routes.md (hunk 2 above)
Comment (Guaris) [reply src/gateway/kong-manager/get-started/services-routes.md:2]: added

File: app/_data/docs_nav_gateway_3.0.x.yml (hunk 4 above)
Comment (Guaris) [reply app/_data/docs_nav_gateway_3.0.x.yml:300]: fixed

File: app/_data/docs_nav_gateway_3.0.x.yml (hunk 13 above)
Comment (Guaris) [reply app/_data/docs_nav_gateway_3.0.x.yml:536]: I did do it intentionally as a placeholder, but I'll fix it here.

File: app/_data/docs_nav_gateway_3.0.x.yml (hunk 14 above)
Comment (Guaris) [reply app/_data/docs_nav_gateway_3.0.x.yml:538]: I did do it intentionally as a placeholder, but I'll fix it here.

**Review by cloudjumpercat — APPROVED** [review]

**Pull request merged**